src/main/java/net/javadiscord/javabot/systems/commands/FormatCodeCommand.java

```diff
@@ -19,25 +19,38 @@
  * Command that allows members to format messages.
  */
 public class FormatCodeCommand implements ISlashCommand, IMessageContextCommand {
+	long id;
+
 	@Override
 	public ReplyCallbackAction handleSlashCommandInteraction(SlashCommandInteractionEvent event) {
 		var idOption = event.getOption("message-id");
 		var formatOption = event.getOption("format");
+		var slashConfig = Bot.config.get(event.getGuild()).getSlashCommand();
 		String format = formatOption == null ? "java" : formatOption.getAsString();
-		long id;
 		if (idOption == null) {
 			if (event.getChannel().hasLatestMessage()) {
-				id = event.getChannel().getLatestMessageIdLong();
+				event.getChannel().getHistory().retrievePast(100) // max limit
```

Comment:
I think fetching 100 messages is a little bit of an overkill.

Reply:
how many do you think we should fetch? 10?

Reply:
how many do you think we should fetch? 10?